api/src/main/resources/workbench-api.yaml

```diff
@@ -5157,6 +5157,77 @@ definitions:
       jwt:
         type: string
         description: the encoded/signed JWT
+  AdminTableUser:
```

Comment:
asiide: I was fooling around with BigQuery yesterday and came up with what I thought was a reasonable way to [break down the monolith](https://pmi-engteam.slack.com/archives/C0150CWF6FP/p1601605839081700) of columns in the user table. It might make sense for our API as well (particularly if types like this could reuse things like the `compliance` section), though you'd have to add mapping annotations in MapStruct.

Another advantage would be reusing descriptions for fields and keeping types in sync (e.g. if we changed userId to `int32`).